app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: List[Player]) -> float:
+    return int(sum(player.get_rating() for player in players))
```

Comment:
There is no need to convert the `sum` output to `int`, leave it as is
```suggestion
    return sum(player.get_rating() for player in players)
```